package.json

```diff
@@ -69,18 +69,18 @@
     "zone.js": "~0.11.5"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^15.1.0-next.3",
-    "@angular-devkit/core": "^15.1.0-next.3",
-    "@angular-devkit/schematics": "^15.1.0-next.3",
-    "@angular/bazel": "^15.1.0-next.3",
+    "@angular-devkit/build-angular": "^15.1.0-rc.0",
+    "@angular-devkit/core": "^15.1.0-rc.0",
+    "@angular-devkit/schematics": "^15.1.0-rc.0",
+    "@angular/bazel": "15.1.0-next.3",
```

Comment:
I tried updating `@angular/bazel` as well, but I couldn't get the build to work, presumably because of the changes mentioned in #26357. One of the errors was because we don't need the `devmode-es2020-bazel.patch`, but after removing it, I had some very vague errors from Bazel like:

```
ERROR: material2/src/material/progress-bar/testing/BUILD.bazel:5:10: for src/material/progress-bar/testing/index.d.ts, previous action: action 'Compiling Angular templates (full prodmode) //src/material/progress-bar/testing:testing', attempted action: action 'Compiling Angular templates (full devmode) //src/material/progress-bar/testing:testing'
ERROR: Analysis of target '//src/material/progress-bar/testing:testing' failed; build aborted: 
INFO: Elapsed time: 51.064s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (284 packages loaded, 13767 targets configured)
error Command failed with exit code 1.
```

Reply:
Yeah, it needs much more changes in this repo to be able to update. It should be fine not updating, or do you need it for TS?

Reply:
I don't need it for the TS update, I tried doing it for consistency.